src/anchorfm-pupeteer/index.js

```diff
@@ -200,6 +187,33 @@ function getSaveDraftOrPublishOrScheduleButtonDescription() {
   };
 }
 
+
+async function clickSelector(page, selector, {visible} = {}) {
+  const options = {};
+  if(visible !== undefined){
+    options.visible = visible;
+  }
+  await page.waitForSelector(selector, options);
+  const element = await page.$(selector);
+  await clickDom(page, element);
+}
+
+async function clickXpath(page, xpath, {timeout} = {}){
```

Comment:
II would advise the third argument to be an "options" argument.

So make the third argument options.